internal/modify/replace_addr_test.go

```diff
@@ -38,60 +39,71 @@ func testReplaceAddr(t *testing.T, modName string) {
-	test(`"\"test @ test\""@example.com`, `"\"b @ b\""@example.com`,
-		map[string]string{`"\"test @ test\""`: `"\"b @ b\""`})
-	test("TeSt@eXAMple.com", "test@example.org",
-		map[string]string{"test@example.com": "test@example.org"})
-	test("test@example.com", "test2@example.com",
-		map[string]string{"test": "test2"})
-	test("test@example.com", "test2@example.org",
-		map[string]string{"test": "test2@example.org"})
-	test("postmaster", "test2@example.org",
-		map[string]string{"postmaster": "test2@example.org"})
-	test("TeSt@examPLE.com", "test2@example.com",
-		map[string]string{"test": "test2"})
-	test("test@example.com", "test3@example.com",
-		map[string]string{
-			"test@example.com": "test3@example.com",
-			"test":             "test2",
+	test("test@example.org", []string{"test@example.org"}, nil)
+	test("postmaster", []string{"postmaster"}, nil)
+	test("test@example.com", []string{"test@example.org"},
+		map[string][]string{"test@example.com": []string{"test@example.org"}})
+	test(`"\"test @ test\""@example.com`, []string{"test@example.org"},
+		map[string][]string{`"\"test @ test\""@example.com`: []string{"test@example.org"}})
+	test(`test@example.com`, []string{`"\"test @ test\""@example.org`},
+		map[string][]string{`test@example.com`: []string{`"\"test @ test\""@example.org`}})
+	test(`"\"test @ test\""@example.com`, []string{`"\"b @ b\""@example.com`},
+		map[string][]string{`"\"test @ test\""`: []string{`"\"b @ b\""`}})
+	test("TeSt@eXAMple.com", []string{"test@example.org"},
+		map[string][]string{"test@example.com": []string{"test@example.org"}})
+	test("test@example.com", []string{"test2@example.com"},
+		map[string][]string{"test": []string{"test2"}})
+	test("test@example.com", []string{"test2@example.org"},
+		map[string][]string{"test": []string{"test2@example.org"}})
+	test("postmaster", []string{"test2@example.org"},
+		map[string][]string{"postmaster": []string{"test2@example.org"}})
+	test("TeSt@examPLE.com", []string{"test2@example.com"},
+		map[string][]string{"test": []string{"test2"}})
+	test("test@example.com", []string{"test3@example.com"},
+		map[string][]string{
+			"test@example.com": []string{"test3@example.com"},
+			"test":             []string{"test2"},
 		})
-	test("rcpt@E\u0301.example.com", "rcpt@foo.example.com",
-		map[string]string{
-			"rcpt@\u00E9.example.com": "rcpt@foo.example.com",
+	test("rcpt@E\u0301.example.com", []string{"rcpt@foo.example.com"},
+		map[string][]string{
+			"rcpt@\u00E9.example.com": []string{"rcpt@foo.example.com"},
 		})
-	test("E\u0301@foo.example.com", "rcpt@foo.example.com",
-		map[string]string{
-			"\u00E9@foo.example.com": "rcpt@foo.example.com",
+	test("E\u0301@foo.example.com", []string{"rcpt@foo.example.com"},
+		map[string][]string{
+			"\u00E9@foo.example.com": []string{"rcpt@foo.example.com"},
 		})
+	
+	if modName == "modify.replace_rcpt" {
+		//multiple aliases
+		test("test@example.com", []string{"test@example.org", "test@example.net"},
```

Comment:
Looks like you got a few formatting inconsistencies there and there.
Could you apply `gofmt` to your code?

Reply:
done - I only applied it to the files I modify but there are others where `gofmt`proposes changes to the headers. I did not touch those to avoid adding to the PR.